webapp-django/crashstats/crashstats/templates/crashstats/daily.html

```diff
@@ -21,6 +21,9 @@
 <div id="mainbody">
 
 <div class="page-heading">
+    <p class="new-daily-report-link">
```

Comment:
You can use the ``old-new-report-link`` class here instead, and then remove the new styling from ``daily.less``.